_posts/2021-10-25-waku-v1-vs-waku-v2.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+name:  "Waku v1 vs Waku v2: bandwidth comparison"
+title:  "Waku v1 vs Waku v2: bandwidth comparison"
+date:   2021-10-25 17:00:00 +0200
+author: hanno
+published: true
+permalink: /waku-v1-vs-waku-v2
```

Comment:
/waku-v1-v2-bandwidth-comparison

Maybe? This is specifically about BW I suppose

Reply:
Done in 21749a4